src/test/scala/uk/gov/hmrc/test/ui/pages/GuaranteeBalance/EoriNumberPage.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.pages.GuaranteeBalance
+
+import uk.gov.hmrc.test.ui.pages.BasePage
+
+object EoriNumberPage extends BasePage {
```

Comment:
Could this not extend `StringPage`, with `override def title(args: String*): String = "What is your EORI number?"`, then in `GuranteeBalanceStepDef` you can call `EoriNumberPage.loadPage()`? Just keeps it consistent with the other pages

Reply:
Ah I see why, because the service title is different to the one in departures